src/docs/asciidoc/reference/manage/application-access-control.adoc

```diff
@@ -0,0 +1,12 @@
+:relative-path: ../../../
+include::{docdir}/variables.adoc[]
+
+As link:#manage-organization[seen previously], an *organization* can have several team or/and several applications. So we need to have the possibility to manage the access control per application.
+
+== Grant access rights
+
+NOTE: This feature is not already done. This will be done soon. Now, you can link:https://www.zetapush.com/contact/[contact us] to manage your access rights for you.
```

Comment:
Use note and tip conventions:
```
.Title of the note (here it could be Contact us to manage access rights)
[NOTE]
====
content of the note
====
```